### DISCUSSION TITLE
Easyeffect Sink / Source volume has no effect

### DISCUSSION THREAD

**Discussion by PhlegmaTREEc** [wwmm/easyeffects#1845] [Q&A] [answered]
Hi, quick question. Changing volume(same for mute) of EE Sink or EE Source has no effect on actual volume. Is it possible to do it (control volume globally)?
I'm on ArcoLinux, Qtile WM
Pipewire, wireplumber
Everything updated

**Comment by wwmm** [comment, marked as answer]
As the volume and mute control were being a source of problems for many users we do not enable them anymore. You can read more information at https://github.com/wwmm/easyeffects/issues/1817.

**Reply by PhlegmaTREEc** [reply]
Damn, I'm sorry, I swear I tried to look for existing topic, but nothing came up, but I just realized what I did wrong there. Really sorry for duplicate

**Reply by wwmm** [reply]
No problems
